packages/cdk/utils/src/replace-units/replace-units.ts

```diff
@@ -0,0 +1,112 @@
+import { Nullable } from '@angular-ru/cdk/typings';
+
+import { isNil } from '../is-nil';
+import { isNotNil } from '../is-not-nil';
+import { UnitsMap } from './types';
+
+const REGEXP_NUMBER: string = `[0-9]+([\.][0-9]+)?`;
```

Comment:
@Schirbak we don't have global regexp constant from `@angular-ru/cdk/regexp`?

Reply:
removed to regexp constants